dmutils/content_loader.py

```diff
@@ -402,7 +402,10 @@ def _get_single_question_data(self, form_data):
                 "assurance": form_data.get(self.id + '--assurance'),
             }
 
-        return {self.id: value if value else None}
```

Comment:
Checkboxes and lists don't get passed through at all if none are checked. This does mean we currently don't have a way of unchecking the last checkbox.